x-pack/plugins/global_search/server/routes/integration_tests/find.test.ts

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { of, throwError } from 'rxjs';
+import supertest from 'supertest';
+import { UnwrapPromise } from '@kbn/utility-types';
```

Comment:
a side note: Finally we will get rid of this https://github.com/microsoft/TypeScript/issues/27711